tendrl_node_agent/monitoring/constants.py

```diff
@@ -0,0 +1,18 @@
+import platform
```

Comment:
I feel, instead this could be a conf file as a yaml/json and load for reference in code. Processing josn/yaml in python is easy :)